cmd/trigger/app/trigger.go

```diff
@@ -66,6 +66,7 @@ func Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	migration.DiscoveryClient.UseLegacyDiscovery = true
```

Comment:
Why is this necessary? Shouldn't content-type negotiation handle compatibility with apiservers that don't support aggregation?

Reply:
I needed this here to ensure I [get the `StorageVersionHash` field populated](https://github.com/kubernetes/enhancements/blob/98e89c38b263e43755d3b8cd20e4083f9f71af02/keps/sig-api-machinery/3352-aggregated-discovery/README.md?plain=1#L782-L787).